app/src/models/uncommitted-changes-strategy.ts

```diff
@@ -1,5 +1,15 @@
-export enum UncommittedChangesStrategy {
+import { IStashEntry } from './stash-entry'
+
+export enum UncommittedChangesStrategyKind {
   askForConfirmation,
```

Comment:
Finally time to update these to be SentenceCase?

Reply:
Not sure why this was resolved, but it looks like the enum was renamed to be `lowerCamelCase` which is different to our other enums 🤔